### DISCUSSION TITLE
'_relevance' does not exist in type

### DISCUSSION THREAD

**Discussion by anthony-bernardo** [prisma/prisma#22587] [Q&A] [answered]
### Question

I'm trying to use relevance in full text search, but the keyword is not recognize either by : 

1. typescript ('_relevance' does not exist in type 'ProductOrderByWithRelationInput | ProductOrderByWithRelationInput[]') o
2. when calling the API route (unknown argument `_relevance)

### How to reproduce (optional)

```
   const products = await prisma.product.findMany({
            take: 10,
            orderBy: {
                _relevance: {
                    fields: ['bio'],
                    search: 'developer',
                    sort: 'asc',
                },
            },
        })
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by nurul3101** [comment, marked as answer]
Hello @anthony-bernardo 👋 

Have you included `fullTextSearch` in the preview feature in the `schema.prisma` file like this?
```prisma
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["fullTextSearch"]
}
```

I just tested a similar code snippet locally, and it works as expected for me. I don't get any TS errors. Have you invoked `npx prisma generate` command after adding the `fullTextSearch` preview feature in the schema file?

<img width="607" alt="Screenshot 2024-01-08 at 5 28 32 PM" src="https://github.com/prisma/prisma/assets/13049676/5ea0e6eb-dbba-4301-ac9e-0f319a2f6206">

**Reply by MarieCHal** [reply]
Thank you! worked for me I was missing the "npx prisma generate" after adding the fullTextSearch.

**Reply by nurul3101** [reply]
I am glad to hear that @MarieCHal 👍 
@anthony-bernardo Is the issue resolved for you as well?

**Reply by anthony-bernardo** [reply]
Thank you for your quick answer! It's working now, thank you ! ✨

**Reply by nurul3101** [reply]
I am glad to hear that! 😄
